obi_one_v2/ec2_ecs_user_data.sh

```diff
@@ -25,32 +26,40 @@ install_s3_mount() {
     echo "install_s3_mount"
     curl -LO https://s3.amazonaws.com/mountpoint-s3-release/latest/x86_64/mount-s3.rpm || return 1
     yum install -y ./mount-s3.rpm || return 1
-    mkdir -p ${mounted_volume_host_path} || return 1
 }
 retry install_s3_mount
 
 # https://github.com/awslabs/mountpoint-s3/issues/441#issuecomment-1676918612
-echo "setup systemd service"
-cat << EOF > /etc/systemd/system/mountpoint-s3.service
+%{ for cfg in mount_buckets ~}
+echo "Setup mountpoint and systemd service for ${cfg.volume_name}"
+
+MOUNT_DIR="${mount_base_dir}${cfg.volume_host_path}"
+SERVICE="mountpoint-s3-${cfg.volume_name}.service"
+mkdir -p "$MOUNT_DIR"
+
+cat << EOF > "/etc/systemd/system/$SERVICE"
 [Unit]
-Description=Amazon S3 mount
+Description=Amazon S3 mount [${cfg.volume_name}]
 Wants=cloud-init.target
```

Comment:
I vaguely remember that I had issues sometimes w/ the sonata cell position service, and I think it was related to the systemd dependencies - which I know nothing about how they really work - see the comment starting line 63.

Something you should be aware of, but it all looks reasonable to me.